packages/core/protocols/src/proto/dxos/halo/credentials.proto

```diff
@@ -233,6 +233,9 @@ message SpaceRecord {
   /// Key of the writable feed for data messages.
   ///   Keyring must have the associated private key.
   dxos.keys.PublicKey write_data_feed_key = 4;
+  
```

Comment:
can you please move these defs into metadata.proto, and reconcile with what's already there?